clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/config/AppengineConfigurationProperties.groovy

```diff
@@ -43,6 +43,10 @@ class AppengineConfigurationProperties {
     String sshKnownHostsFilePath
     boolean sshTrustUnknownHosts
     GcloudReleaseTrack gcloudReleaseTrack
+    List<String> services
+    List<String> versions
```

Comment:
If I've blacklisted `myapp-v001`, does that mean that `myapp-v002` will still show up? 

I think it might be more intuitive to whitelist / blacklist using regex so that I could say that I want to exclude any version that starts with `myapp`. You could also whitelist / blacklist using app name (and use Frigga to parse the server group's name).

Reply:
:+1:

Reply:
✅ done & doc strings updated here: https://github.com/spinnaker/halyard/pull/1102/files